### PR TITLE
Don't assume everything is English Wikipedia (T145436)

### PR DESCRIPTION
**Pull request by samwilson**
To resolve this task: https://phabricator.wikimedia.org/T145436

### CODE REVIEW

File: public_html/i18n/fr.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+	"name": "CopyPatrol",
+	"wikiprojects": "WikiProjets",
```
Comment (Niharika29) [inline public_html/i18n/fr.json]: Wikiprojects? :)

Comment (samwilson) [inline public_html/i18n/fr.json]: Yup! It's French: https://fr.wikipedia.org/wiki/Aide:Projet

But it's a good point: I didn't translate 'CopyPatrol'... is it "CopiePatrouille"? Or "patrouille du droit d'auteur" as google says?... :-) We need a French speaker here...

Comment (samwilson) [inline public_html/i18n/fr.json]: Should we localise the logo? (The text bit of it, I mean.)

**Comment by samwilson** [comment]
I've squashed all this into one commit, #34. Closing this one.

**Pull request closed without merging**